tech.json

```diff
@@ -908,6 +908,17 @@
           "note": [
             "Starting from an air environment, run into water and charge a spark in the water. This is typically done while crossing a door transition from a room with air physics to a room with water.",
             "This is easy to do; the tech just represents knowledge that it is possible."
+          ],
+          "extensionTechs": [
+            {
+              "name": "canStutterWaterShineCharge",
+              "requires": [ "canWaterShineCharge" ],
+              "note": [
+                "Performing a canWaterShineCharge in a shorter runway by using a stutter.",
+                "Perform a stutter shortly before the water entrance, typically a door transition, by briefly releasing and then repressing forward. This allows Samus to enter the water with little momentum.",
```

Comment:
...repressing forward before, but as close to, the transition as possible while continuing the animation cycle.
This carries the stutter into the water where Samus will run in place while building the shinecharge.